docs/kubernetes/user-guides/driver-install.md

```diff
@@ -1,22 +1,53 @@
 # Kubernetes Driver Installation Guide
 
+## GCP Permissions Required
+
+The service account running `setup-project.sh` in order to set up a service
```

Comment:
The meta wording is a little confusing. How about something like "The setup-project.sh script only needs to be run once per project to generate a service account for the driver. The user or service account running this script needs the following permissions"

Reply:
yeah this is really hard to work. I've updated it all, PTAL